core/json/src/main/scala/net/liftweb/json/JsonAST.scala

```diff
@@ -118,7 +118,7 @@ object JsonAST {
      * nothing is found, you'll get a `JNothing`.
      */
     def \(nameToFind: String): JValue = {
-      findDirectByName(List(this), nameToFind) match {
+      findDirectByName(this :: Nil, nameToFind) match {
```

Comment:
I'm debating about whether or not this is an obscure enough change to document somehow. I'll keep thinking on it.

Reply:
I think it's worth leaving a note here, in the style of “Use :: instead of List() to avoid an extra array allocation.”

Reply:
done